src/k8s/pkg/k8sd/api/node_access_handler_test.go

```diff
@@ -0,0 +1,103 @@
+package api
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/canonical/k8s/pkg/snap"
+	"github.com/canonical/k8s/pkg/snap/mock"
+	. "github.com/onsi/gomega"
+)
+
+func TestValidateNodeTokenAccessHandler(t *testing.T) {
+	const tokenFileName = "tmp-token-file"
+	const tokenHeaderName = "X-Node-Token"
```

Comment:
```suggestion
	const (
		tokenFileName   = "tmp-token-file"
		tokenHeaderName = "X-Node-Token"
	)
```

Reply:
I was actually thinking that we can centralize this `tokenHeaderName` (`node-token`) to be used in both `endpoints.go` and in this test. WDYT?

Reply:
Done